kubernetes/examples/FasterTransformer_HuggingFace_Bert.md

```diff
@@ -0,0 +1,279 @@
+# Faster Transformer HuggingFace Bert example in Kubernetes Torchserve.
```

Comment:
Hamid already has an example with a similar name https://github.com/pytorch/serve/tree/master/examples/FasterTransformer_HuggingFace_Bert

Perhaps we can rename this tutorial HuggingFace Faster Transformer Kubernetes or expand on the tutorial @HamidShojanazeri  has